app/views/contexts/summary/list_collector_block.py

```diff
@@ -128,6 +134,21 @@ def list_summary_element(self, summary: Mapping[str, Any]) -> dict[str, Any]:
             **list_summary_context,
         }
 
+    def repeating_block_question_element(self, block: ImmutableDict) -> list[dict]:
```

Comment:
I think the naming is not quite right. Should this just not be something like `get_repeating_block_answers_blocks`. I think the existing `get_releated_answers` is misleading. It implies you get back a list of answers. We should rename to something like `get_related_answer_blocks_by_list_item`.

Reply:
One day I'll master naming variables 🙃 , definitely prefer those suggestions 👍  , updated answers as suggested and swapped this one to `get_repeating_block_related_answer_blocks` to keep inline with the second (but can go with original too, dont mind)

Reply:
Happy with the change 👍